### DISCUSSION TITLE
How to use axios to send windows credentials?

### DISCUSSION THREAD

**Discussion by Microkiller** [axios/axios#5603] [Q&A]
I am trying to call an API that uses windows authentication. 
I tried to use 
axios.defaults.withCredentials = true

But nothing is in the hearder authentication.
How can I pass the windows authentication using axios?

**Comment by jasonsaayman** [comment]
`withCredentials` does not create a Windows/NTLM/Kerberos `Authorization` header. It only tells the browser to include credentials that the browser already manages, such as cookies or HTTP auth credentials, when CORS allows it.

For Windows Authentication:
- In the browser, Axios cannot manually send the current Windows credentials. The browser controls Integrated Windows Authentication. The API must be in a trusted/intranet zone, CORS must allow credentials, and the browser/server must negotiate `Negotiate`/`NTLM`.

- In Node.js, Axios does not implement NTLM/Kerberos itself. You would need a dedicated agent/library that supports the auth scheme, or call through a backend/proxy that handles Windows auth.
So this is not something `withCredentials` alone can enable.
